src/plugins/ui_actions/public/triggers/select_range_trigger.ts

```diff
@@ -22,6 +22,6 @@ import { Trigger } from '.';
 export const SELECT_RANGE_TRIGGER = 'SELECT_RANGE_TRIGGER';
 export const selectRangeTrigger: Trigger<'SELECT_RANGE_TRIGGER'> = {
   id: SELECT_RANGE_TRIGGER,
-  title: 'Select range',
+  title: '',
```

Comment:
I've added a comment in the code.

It is empty so that when `ui_actions` context menu is rendered it does not render a title.

I did not add a new field to `Trigger` interface like `showTitle?: boolean` as I thought it is out of scope of this PR. But, in general, I think `Trigger` could extend `Presentable` and maybe it could have an extra field like `showTitle?: boolean` that determines if context menu should have a title. Or, simply like now if `.getDisplayName()` is falsy, don't show a title.